src/schema/manifest-schema.json

```diff
@@ -18,13 +18,13 @@
                             "default": "42a1",
                             "description": "Minimum version of Gecko to support. Defaults to '42a1'. (Requires Gecko 45)",
                             "type": "string",
-                            "pattern": "^[0-9]{1,3}(.[a-z0-9]+)+$"
+                            "pattern": "^[0-9]{1,3}(\\.[a-z0-9]+)+$"
```

Comment:
It's in JSON where \ is a valid escape character, so you have to do \\ in the regex, which is annoying.